platform/test/spec/testActivityManagerSpec.js

```diff
@@ -6,91 +6,67 @@ import { ActivityConfigValidator } from "../../src/ActivityConfigValidator.js";
-        const refPanelDef = () => {};
+            spySearch = spyOn( utility, "getWindowLocationSearch");
 
-        // Call the target object
-        var am = new ActivityManager(refPanelDef , fileh);
+            // Call the target object
+            am = new ActivityManager(refPanelDef, fileh);
+        })
 
         // Check the expected results
-        expect(am.accessPanelDef).toBe(refPanelDef);
-    })  
-
-    it("constructor -  initialises the fileHandler property  using param fileHandler", () => {
-        // Setup
-        const fileh = new FileHandler("test://th.url") 
-        const refPanelDef = () => {};
+        it("can be created", () => {
+            expect(am).toBeInstanceOf(ActivityManager);
+        })
 
-        // Call the target object
-        var am = new ActivityManager(refPanelDef , fileh);
+        it("initialises the configValidator property", () => {
+            expect(am.configValidator).toBeInstanceOf(ActivityConfigValidator);
+        })        
 
-        // Check the expected results
-        expect(am.fileHandler).toBe(fileh);
-    })  
+        it("initialises the accessPanelDef property using param panelDefAccessor", () => {
+            expect(am.accessPanelDef).toBe(refPanelDef);
+        })  
 
-    it("constructor -  uses the query string to set the activitiesUrl property", () => {
-        // Setup
-        const ACTIVITY_URL="test://a.url";
-        const QUERY = "?activities=" + ACTIVITY_URL;
-        const fileh = new FileHandler("test://th.url") 
-        const refPanelDef = () => {};
+        it("initialises the fileHandler property  using param fileHandler", () => {
+            expect(am.fileHandler).toBe(fileh);
+        })  
 
-        spyOn( utility, "getWindowLocationSearch").and.returnValue(QUERY);
+        it("uses the query string to set the activitiesUrl property", () => {
+            const QUERY = "?activities=" + ACTIVITY_URL;
+            spySearch.and.returnValue(QUERY); //Update the spy
 
-        // Call the target object
-        var am = new ActivityManager(refPanelDef , fileh);
+            am = new ActivityManager(refPanelDef, fileh);
 
-        // Check the expected results
-        expect(utility.getWindowLocationSearch).toHaveBeenCalled();
-        expect(am.activitiesUrl).toBe(ACTIVITY_URL);
-    })
+            expect(utility.getWindowLocationSearch).toHaveBeenCalled();
+            expect(am.activitiesUrl).toBe(ACTIVITY_URL);
+        })
 
-    it ("constructor -  if the current activity is provided in the url query string, set the activityId property", () => {
-        // Setup
-        const ACTIVITY_URL="test://a.url";
-        const ACTIVITY_ID="a1"
-        const QUERY = "?"+ ACTIVITY_ID + "=&" + "activities=" + ACTIVITY_URL;
-        const fileh = new FileHandler("test://th.url") 
-        const refPanelDef = () => {};
+        it ("if the current activity is provided in the url query string, set the activityId property", () => {
```

Comment:
```suggestion
        it ("sets the activityId property if the current activity is provided in the url query string", () => {
```

Reply:
Updated in c1fc48ec710862c2619fcad8fc297215a00334e8